src/main/java/io/reactivex/exceptions/OnCompleteFailedException.java

```diff
@@ -18,7 +18,11 @@
     private static final long serialVersionUID = -6179993283427447098L;
 
     public OnCompleteFailedException(Throwable cause) {
-        super(cause);
+        super(cause != null ? cause : new NullPointerException());
```

Comment:
ok, but I though it's ok to have null as cause…

On Tue, Jul 19, 2016 at 12:35 AM, David Karnok <notifications@github.com>
wrote:

> In src/main/java/io/reactivex/exceptions/OnCompleteFailedException.java
> <https://github.com/ReactiveX/RxJava/pull/4204#discussion_r71234794>:
>
> > @@ -18,7 +18,11 @@
> >      private static final long serialVersionUID = -6179993283427447098L;
> >
> >      public OnCompleteFailedException(Throwable cause) {
> > -        super(cause);
> > +        super(cause != null ? cause : new NullPointerException());
>
> This has been discussed back then: instead of crashing the code further,
> it sets an NPE with the current stack trace to give something to the
> receiver.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/4204/files/a9752a3d697c31ad1ac46bb41891777038e0b0d8#r71234794>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA7B3K3Dpwpx9tC1aBXSB4f_FR2yi0iCks5qW_GjgaJpZM4JMU7W>
> .
>